CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [v5.8.0](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.0) (2022-02-28)
+## [v5.8.1](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.1) (2022-04-07)
+[Full Changelog](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v5.7.2...v5.8.0)
+
+## Changed
+
+- Update agent version from v3.35.0 to v3.35.1 [#1005](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/1005) (@moskyb)
+
+## [v5.8.0](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.0) (2022-03-28)
```

Comment:
Was the v5.8.0 release date wrong before?

Reply:
yeah, we really had the B Team on that release (me)